students/aravichander/lesson08/circle_class.py

```diff
@@ -0,0 +1,54 @@
+from math import pi 
+circlelist = []
+
+#Creating class to represent circle object 
+class Circle:
+	def __init__(self,radius):
```

Comment:
use four spaces to indent -- always! 

I think these are tabs. Make sure your editor is configured to put in four spaces when you hit the <tab> key.